src/content/docs/apm/agents/go-agent/installation/install-automation-new-relic-go.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Simplify installation with Go easy instrumentation'
+tags:
+  - Agents
+  - Go agent
+  - Installation
+metaDescription: How to use a script to get code suggestions to instrument your Golang apps and services.
+freshnessValidatedDate: 2024-08-20 
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+Go is a compiled language with an opaque runtime, making it unable to support automatic instrumentation like other languages. For this reason, the New Relic Go agent is designed as an SDK. Since the Go agent is an SDK, it requires more manual work to set up than agents for languages that support automatic instrumentation.
+
+In an effort to reduce that manual effort, the Go agent team created the Go easy instrumentation tool, which is currently in preview. This tool will do most of the work for you by suggesting changes to your source code that instrument your application with the New Relic Go agent.
+
+To get started, check out this four-minute video, or skip down to [How it works](#how-it-works):
+
+[![asciicast](https://asciinema.org/a/r0Il7o2eMiZaLKHIlew3IL2nx.svg)](https://asciinema.org/a/r0Il7o2eMiZaLKHIlew3IL2nx)
+
+## Preview notice [#preview-notice]
+
+This feature is currently provided as part of a product preview and is subject to our New Relic Experimental policies. Recommended code changes are suggestions only and should be subject to human review for accuracy, applicability, and appropriateness for your environment. This feature should only be used in non-critical, non-production environments that do not contain sensitive data.
```

Comment:
Can be added as an important call out

Reply:
@cbehera-newrelic I'll check with the legal team to see what they want to do.

Reply:
@rhetoric101 Do we need this full statement? I almost want to just stick this into the callout at the very top. It's pretty trivial to scroll past. 

Here, it really disrupts the flow of your doc, either as a callout or in its current state.

Here's an idea: what if you put it into a collapser?

Reply:
I think that we would want legal sign off for that. I am not sure how prominently the disclaimer needs to be displayed.

Reply:
@iamemilio I received permission from Legal to put this "Preview notice" section in a collapser. I've just pushed up this change.